services/queue/src/api.js

```diff
@@ -923,12 +920,7 @@ builder.declare({
     workerType,
   });
 
-  const worker = await this.Worker.load({
-    provisionerId,
-    workerType,
-    workerGroup,
-    workerId,
-  }, true);
+  const worker = await Worker.get(this.db, provisionerId, workerType, workerGroup, workerId, new Date());
```

Comment:
The previous query did not consider `expires`. Should it be set  to `null`? This comment applies to L1031 as well.

Reply:
I think we've started to try to filter out expired results from the api so it makes sense to do here?